### PR TITLE
Bug 1920911: Do not include DVM volumes in stage backups

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: pkg/controller/migmigration/task.go [hunk 1]
```diff
@@ -1337,12 +1337,19 @@ func (t *Task) getDestinationClient() (compat.Client, error) {
 }
 
 // Get the persistent volumes included in the plan which are not skipped.
-func (t *Task) getPVs() migapi.PersistentVolumes {
+// This function will only return PVs that are being copied via restic or snapshot
+func (t *Task) getStagePVs() migapi.PersistentVolumes {
```
Comment (alaypatel07) [inline pkg/controller/migmigration/task.go:1341]: can we name this `getResticOrSnapshotPVs`? sorry but took me a while to figure out `Stage` means `ResticOrSnapshot`and then saw it in the comments

**Review by alaypatel07 — APPROVED** [review]
I wanted to understand how bad writing unit tests would be and it was not that bad

https://github.com/alaypatel07/mig-controller/commit/ab7359b648ce084f578f3c64f8d7b42c5ee4d866 if you think it makes sense please open the PR to your branch and merge it to get the unit tests here. After writing them it makes me far more confident in approving this PR :)

File: pkg/controller/migmigration/task.go (hunk 1 above)
Comment (dymurray) [reply pkg/controller/migmigration/task.go:1341]: @alaypatel07 this includes `Move` PVs as well which is why I included `Stage` in the name. I.e. any PV that needs to be included in a stage backup.

What do you think?

Comment (dymurray) [reply pkg/controller/migmigration/task.go:1341]: I'll add a comment

**Pull request merged**